lib/model/item.dart

```diff
@@ -15,6 +15,12 @@ class Item {
   bool isChecked = false;
 
   Item([this.value = '', this.itemType = ItemType.text]);
+
+  void copyOnto(Item item) {
```

Comment:
It would be good to include an explanation comment above the method. Explain **why** it is needed, not **what** it does.